docs/source/how-to/build-a-catalog-from-timeseries-files.md

```diff
@@ -58,11 +58,11 @@ The only parts of ecgtools we need are the `Builder` object and the `parse_cesm_
 import pathlib
 
 import dask
+import distributed
```

Comment:
don't we need to instantiate a local cluster for this to have an effect?

Reply:
Right - that needs to be removed too